frontend/app/common_env

```diff
@@ -0,0 +1,7 @@
+export LEDGER_CANISTER_ICP=ryjl3-tyaaa-aaaaa-aaaba-cai
```

Comment:
if these things are common why are they not just hard coded? Then the cache invalidation will just work. If we change them, it's just a code change. It's not like we can modify them without a release _anyway_.

Reply:
They can just be constants in the code.

Reply:
Yeah we could hard code them, I just thought it might be easier to keep them all in a common file

Reply:
But they are no less "in a common file" if they are in a constants module. Seems like less unnecessary plumbing.

Reply:
Sorted.